src/modules/previewpane/powerpreview/settings.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include <pch.h>
+#include <string>
+#include "resource.h"
+#include <settings_objects.h>
+#include "registry_wrapper_interface.h"
+
+namespace PowerPreviewSettings
+{
+	// PowerToy Winodws Explore File Preview Settings.
+	class FileExplorerPreviewSettings
+	{
+	private:
+		bool m_isPreviewEnabled;
+		std::wstring m_name;
+		std::wstring m_description;
+		std::wstring m_displayName;
+        RegistryWrapperIface * m_registryWrapper;
+
+		LPCWSTR m_clsid;
+		LPCWSTR m_subKey = L"Software\\Microsoft\\Windows\\CurrentVersion\\PreviewHandlers";
```

Comment:
no reason to store this pointer as a class member.

Reply:
@yuyoyuppe `m_subkey` is used by multiple methods in `EnablePreview` and `DisablePreview`

Reply:
it doesn't make any difference, since it's a constant pointer, we could just make it `static` member.